internal/mock/cloudsql.go

```diff
@@ -75,11 +75,13 @@ func (f FakeCSQLInstance) serverCACert() ([]byte, error) {
 	if f.signer != nil {
 		return f.signer(f.Cert, f.Key)
 	}
+
 	if f.serverCAMode == "" || f.serverCAMode == "GOOGLE_MANAGED_INTERNAL_CA" {
 		// legacy server mode, return only the server cert
 		return toPEMFormat(f.certs.serverCert)
 	}
 	return toPEMFormat(f.certs.casServerCertificate, f.certs.serverIntermediateCaCert, f.certs.serverCaCert)
+
```

Comment:
nit: let's omit these changes from the PR since they're unrelated

Reply:
Fixed.